cmip6_downscaling/methods/bcsd/tasks.py

```diff
@@ -60,18 +62,36 @@ def calc_spacial_anomalies(
 ) -> UPath:
     target = (
         intermediate_dir
-        / "interpolate_obs"
+        / "spatial_anomalies"
         / "{obs}_{model}_{variable}_{latmin}_{latmax}_{lonmin}_{lonmax}_{train_dates[0]}_{train_dates[1]}".format(
             **asdict(run_parameters)
         )
     )
-    if use_cache and (target / '.zmetadata').exists():
-        print(f'found existing target: {target}')
+    if use_cache and (target / ".zmetadata").exists():
+        print(f"found existing target: {target}")
         return target
 
-    # TODO
+    # take interpolated obs and rechunk into full_time
+    coarse_obs_interpolated_rechunked_path = rechunk_zarr_array_with_caching(
+        interpolated_obs_path, chunking_approach="full_time", max_mem="2GB"
+    )
+
+    # original obs rechunked into full-time
+    obs_rechunked_path = rechunk_zarr_array_with_caching(
+        obs_path, chunking_approach="full_time", max_mem="2GB"
+    )
```

Comment:
let's move both of these rechunk calls into stand-alone tasks (called in flow.py)